package.py

```diff
@@ -93,6 +99,74 @@ def __init__(self) -> None:
         if os.environ.get('ROTKI_VERSION') is None:
             os.environ.setdefault('ROTKI_VERSION', self.rotki_version)
 
+        self.__certificate_mac = os.environ.get(MAC_CERTIFICATE)
+        self.__certificate_win = os.environ.get(WIN_CERTIFICATE)
+        self.__csc_password = os.environ.get(CERTIFICATE_KEY)
+        self.__appleid = os.environ.get(APPLE_ID)
+        self.__appleidpass = os.environ.get(APPLE_ID_PASS)
+
+        os.environ.pop(MAC_CERTIFICATE, None)
+        os.environ.pop(WIN_CERTIFICATE, None)
+        os.environ.pop(CERTIFICATE_KEY, None)
+        os.environ.pop(APPLE_ID, None)
+        os.environ.pop(APPLE_ID_PASS, None)
+
+    @staticmethod
+    def temp_dir() -> str:
```

Comment:
There is no need to do this if you just want the system's temp. Python has a cross platform way to do this in the stdlib.